assets/xml/objects/object_um.xml

```diff
@@ -10,26 +10,29 @@
         <Texture Name="object_um_Tex_002148" OutName="tex_002148" Format="rgba16" Width="32" Height="32" Offset="0x2148" />
         <Texture Name="object_um_Tex_002948" OutName="tex_002948" Format="rgba16" Width="32" Height="32" Offset="0x2948" />
         <Texture Name="object_um_Tex_003148" OutName="tex_003148" Format="rgba16" Width="32" Height="32" Offset="0x3148" />
-        <DList Name="object_um_DL_003C60" Offset="0x3C60" />
+        <DList Name="gUmMinigamePotDL" Offset="0x3C60" />
```

Comment:
I would prefer to avoid calling them "milk-bottles" because it could be confused with actual bottles of milk item.
Would it be ok to add a comment explaining what those are?
```suggestion

        <!-- The bottles which the bandits try to destroy in the milk-run minigame. Referred as "milk-bottles" by Cremia in-game. -->
        <DList Name="gUmMinigamePotDL" Offset="0x3C60" />
```

Reply:
Sure, that works for me.